tox.ini

```diff
@@ -15,17 +15,16 @@
 # "tox -e py38-macos-pyqt" will test python3.8 with pyqt on macos
 # (even if a combination of factors is not in the default envlist
 # you can run it manually... like py39-linux-pyside2)
-envlist = py{38,39,310,311}-{linux,macos,windows}-{pyqt5,pyside2,pyqt6,pyside6,headless}-{cov,no_cov},mypy
+envlist = py{39,310,311,312}-{linux,macos,windows}-{pyqt5,pyside2,pyqt6,pyside6,headless}-{cov,no_cov},mypy
 isolated_build = true
 toxworkdir={env:TOX_WORK_DIR:/tmp/.tox}
 
 [gh-actions]
 python =
-    3.8: py38
     3.9: py39
-    3.9.0: py390
```

Comment:
Why this removal?

Reply:
Because we do not have any py390 job in tox